mollie/api/client.py

```diff
@@ -41,38 +44,49 @@
 from .resources.subscriptions import Subscriptions
 from .version import VERSION
 
+if TYPE_CHECKING:
+    from .typing import Timeout
 
-class Client(object):
-    CLIENT_VERSION = VERSION
-    API_ENDPOINT = "https://api.mollie.com"
-    API_VERSION = "v2"
-    UNAME = " ".join(platform.uname())
 
-    OAUTH_AUTHORIZATION_URL = "https://www.mollie.com/oauth2/authorize"
-    OAUTH_AUTO_REFRESH_URL = API_ENDPOINT + "/oauth2/tokens"
-    OAUTH_TOKEN_URL = API_ENDPOINT + "/oauth2/tokens"
+class Client(object):
+    CLIENT_VERSION: str = VERSION
+    API_ENDPOINT: str = "https://api.mollie.com"
+    API_VERSION: str = "v2"
+    UNAME: str = " ".join(platform.uname())
+
+    OAUTH_AUTHORIZATION_URL: str = "https://www.mollie.com/oauth2/authorize"
+    OAUTH_AUTO_REFRESH_URL: str = API_ENDPOINT + "/oauth2/tokens"
+    OAUTH_TOKEN_URL: str = API_ENDPOINT + "/oauth2/tokens"
+
+    access_token: Union[dict[Any, Any], None]
+    api_key: Union[str, None]
```

Comment:
```suggestion
    api_key: Optional[str]
```

The typing docs state that 
> Optional[X] is equivalent to X | None (or Union[X, None])

Did you made an conscious choice for the `Union[X, None]` variant?

Reply:
Updated the code, as there is no functional difference. Thanks for the note.